### PR TITLE
Fix: add padding bottom to the app frame

### PR DESCRIPTION
**Pull request by mikhailxyz**
## What it solves

Resolves https://github.com/safe-global/safe-react-apps/issues/528

## How this PR fixes it

It adds a padding-bottom to the app iframe so it can be scrolled down

## How to test it

Use this pathname `rin:0x91dB860c37E83dab0A4A005E209577E64c61EEfA/apps?appUrl=https%3A%2F%2Fsafe-apps.dev.gnosisdev.com%2Fdrain-safe` with the PR deployment and scroll to the bottom

## Screenshots

image
image

### CODE REVIEW

**Review by iamacook — COMMENTED** [review]
Looks good. Could you also square off the bottom corners of the queue bar?

image

**Comment by DaniSomoza** [comment]
It seems that if you remove the Bar, the padding is still present right?:

https://user-images.githubusercontent.com/26763673/192298651-56f79385-cc20-4435-9f10-dab6d21450fd.mov

**Comment by mikhailxyz** [comment]
> It seems that if you remove the Bar, the padding is still present right?:
> 
>  Grabacion.de.pantalla.2022-09-26.a.las.16.10.08.mov

aww good catch 🙈

File: src/components/safe-apps/AppFrame/TransactionQueueBar/index.tsx [hunk 1]
```diff
@@ -43,23 +43,30 @@ const TransactionQueueBar = (): ReactElement | null => {
               unmountOnExit: true,
               mountOnEnter: true,
             }}
+            sx={{
+              // there are very specific rules for the border radius that we have to override
+              borderBottomLeftRadius: '0 !important',
+              borderBottomRightRadius: '0 !important',
```
Comment (iamacook) [inline src/components/safe-apps/AppFrame/TransactionQueueBar/index.tsx:49]: I believe there is a `square` prop on the `Accordion` that can achieve this.

**Review by iamacook — APPROVED** [review]

File: src/components/safe-apps/AppFrame/TransactionQueueBar/index.tsx (hunk 1 above)
Comment (mikhailxyz) [reply src/components/safe-apps/AppFrame/TransactionQueueBar/index.tsx:49]: But it would unround all the corners, not only the bottom ones, wouldn't it?

Comment (iamacook) [reply src/components/safe-apps/AppFrame/TransactionQueueBar/index.tsx:49]: Valid point.

**Pull request merged**